card/mod/04_settings/lib/card/setting.rb

```diff
@@ -21,7 +21,8 @@ def self.extended(host_class)
       pointer:     "Pointer",
       editing_cue: "Editing cues",
       event:       "Events",
-      other:       "Other"
+      other:       "Other",
+      config:      "Config"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.